src/XIVLauncher.Common/Dalamud/AssetManager.cs

```diff
@@ -225,16 +225,14 @@ private static void CleanUpOld(DirectoryInfo baseDir, int version)
             if (GameHelpers.CheckIsGameOpen())
                 return;
 
-            for (int i = version; i >= version - 30; i--)
+            foreach (DirectoryInfo toDelete in baseDir.GetDirectories())
             {
-                var toDelete = Path.Combine(baseDir.FullName, i.ToString());
-
                 try
                 {
-                    if (Directory.Exists(toDelete))
+                    if (Directory.Exists(toDelete.FullName) && toDelete.Name != "dev" & toDelete.Name != version.ToString())
```

Comment:
Pretty sure that should be a `&&` between `"dev"` and `toDelete.Name`, unless I misunderstand what you're doing here. Although I suppose a bitwise & of two boolean values might have the same result.

Reply:
Ahh beans, you're absolutely right. It definitely works as is, but that's not what is intended. Good catch, this was me not correcting the second `&` after the first not working.